ongebo/fff-frontend
# #163673616 Enable users to login to Fast-Food-Fast

### What does this PR do?
Enables users to log in to Fast-Food-Fast
#### Description of the task to be completed?
- Add action types for user login
- Add reducers for user login
- Update state shape to store information about user login
- Add tests for login reducers
- Define asynchronous action to login a user
- Write tests for action to login a user
- Add component view for user login
- Refactor Header component to a layout folder
- Remove empty initial setup component files
- Connect LoginForm to Redux for dynamic behaviour
- Disable CORS in fetch API requests
#### How should this be manually tested?
- Install all dependencies on this branch and start the server with `yarn start`
- Navigate to /login in the browser and try to log in
- Verify that everything is working correctly
#### What are the relevant Pivotal Tracker stories?
[#163673616](https://www.pivotaltracker.com/story/show/163673616)
